docs/operations/high_availability.md

```diff
@@ -1,96 +1,41 @@
 
-However, when running kubernetes with a single master, if the master fails, the kubernetes API will be unavailable, but pods and services that are running on the (unaffected) nodes should continue to operate.  In this situation, we won't be able to do anything that involves the API (adding nodes, scaling pods, replacing
-terminated pods), and kubectl won't work.  However your application should continue to run, and most applications
-could probably tolerate an API outage of an hour or more.
+If you already have a single-master cluster you would like to convert to a multi-master cluster, read the [single to multi-master](../single-to-multi-master.md) docs.
 
-Moreover, kops runs the masters in an automatic replacement mode.  Masters are run in auto-scaling groups, with
-the data on an EBS volume.  If a master node is terminated, the ASG will launch a new master instance, and kops
-will mount the master volume and replace the master.
 
-In short:
+## Creating a HA cluster
 
-* A single master kops cluster is still reasonably available; if the master instance terminates it will be automatically
-  replaced.  But the use of EBS binds us to a single AZ, and in the event of a prolonged AZ outage, we might experience
-  downtime.
-* A multi-node kops cluster can tolerate the outage of a single AZ
+### Example 1: public topology
 
-
-Using Kops HA
--------------
-
-We can create HA clusters using kops, but only it's important to note that migrating from a single-master
-cluster to a multi-master cluster is a complicated operation (described [here](../single-to-multi-master.md)).
-If possible, try to plan this at time of cluster creation.
-
-When you first call `kops create cluster`, you specify the `--master-zones` flag listing the zones you want your masters
-to run in, for example:
+The simplest way to get started with a HA cluster is to run `kops create cluster` as shown below. The `--master-zones` flag listing the zones you want your masters
+to run in. By default, kops will create one master per AZ. Since the kubernetes etcd cluster runs on the master nodes, you have to specify an odd number of zones in order to obtain quorum.
 
 ```
 kops create cluster \
     --node-count 3 \
     --zones us-west-2a,us-west-2b,us-west-2c \
     --master-zones us-west-2a,us-west-2b,us-west-2c \
-    --node-size t2.medium \
-    --master-size t2.medium \
-    --topology private \
-    --networking kopeio-vxlan \
     hacluster.example.com
 ```
 
-Kubernetes relies on a key-value store called "etcd", which uses the Quorum approach to consistency,
-so it is available if 51% of the nodes are available.
-
-As a result there are a few considerations that need to be taken into account when using kops with HA:
-
-* Only odd number of masters instances should be created, as an even number is likely _less_ reliable than the lower odd number.
-* Kops has experimental support for running multiple masters in the same AZ, but it should be used carefully.
```

Comment:
I feel like this is an important sentence to keep  - the idea that you can repeat the zone.  If you only have 2 AZs in a region, whether it's better to run in 2 AZs or all in one is debatable, I guess - we seem to have suggested at the end that running in one region is best.

(Agree with dropping the federation stuff though!)

Reply:
I thought there was no longer any region with 2 zones anymore, but turns out it is one: Beijing. I'll add a version of that back in.

Reply:
👍  Some regions have more than two AZs but only two of them are effectively usable due to instance type availability: us-west-1 for example.